Services/AdvancedEditing/classes/class.ilObjAdvancedEditingGUI.php

```diff
@@ -150,33 +143,28 @@ public function addSubtabs() : void
     
     public function addPageEditorSettingsSubtabs() : void
     {
-        $ilCtrl = $this->ctrl;
-        $ilTabs = $this->tabs;
```

Comment:
@mjansenDatabay I also like to visualise important dependencies at the beginning of a function and not to hide them. Makes it easier to reorganise things.

Reply:
@alex40724 Should I revert this?

Reply:
No, thanks. Maybe avoid it in the future.